Source/ASNodeController+Beta.m

```diff
@@ -47,6 +47,11 @@ - (instancetype)init
   return self;
 }
 
+- (void)dealloc
+{
+    [_weakNode removeInterfaceStateDelegate:self];
```

Comment:
`-removeInterfaceStateDelegate:` needs to be called on main, and objects are de-allocated off main by default (code [here](https://github.com/TextureGroup/Texture/blob/master/Source/ASMainThreadDeallocation.mm#L147)). Since `NSHashTable` is not thread-safe, we probably have to dispatch to main.

Reply:
@garrettmoon I still think that we should dispatch to main here, no?